tests/TestCase.php

```diff
@@ -48,6 +48,7 @@ abstract class TestCase extends BaseTestCase
 	public const SAMPLE_FILE_GMP_IMAGE = 'tests/Samples/google_motion_photo.jpg';
 	public const SAMPLE_FILE_MONGOLIA_IMAGE = 'tests/Samples/mongolia.jpeg';
 	public const SAMPLE_FILE_NIGHT_IMAGE = 'tests/Samples/night.jpg';
+	public const SAMPLE_FILE_SUNSET_IMAGE = 'tests/Samples/fin de journée.jpg';
```

Comment:
I would prefer to keep that list in alphabetic order. I will push a commit myself.

Reply:
Fixed it myself in https://github.com/LycheeOrg/Lychee/pull/1453/commits/a28747ea885803d6280052f3490283d7429dbccc.